go.mod

```diff
@@ -63,9 +63,9 @@ require (
 	golang.org/x/term v0.15.0
 	golang.org/x/time v0.0.0-20220922220347-f3bd1da661af
 	gonum.org/v1/gonum v0.11.0
-	google.golang.org/genproto/googleapis/rpc v0.0.0-20230711160842-782d3b101e98
-	google.golang.org/grpc v1.58.3
-	google.golang.org/protobuf v1.31.0
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20231002182017-d307bd883b97
+	google.golang.org/grpc v1.60.1
```

Comment:
Updating the `grpc` version has historically been... interesting. I think we'll probably want to hold off on merging this PR until a stable version of `v1.11.x` is released... We may also want to update to `v1.61.0` - as apparently that fixes a regression introduced in `v1.60`.

Reply:
yea sure